blocksuite/affine/inlines/latex/src/latex-node/latex-node.ts

```diff
@@ -140,6 +140,7 @@ export class AffineLatexNode extends SignalWatcher(
               try {
                 katex.render(latex, latexContainer, {
                   displayMode: false,
+                  output: 'mathml',
```

Comment:
we should output both mathml and html to keep the latex display correctly on all platforms.